cmd/glasskube/cmd/install.go

```diff
@@ -356,6 +384,7 @@ func init() {
 		"Specify the name of the package repository to install this package from")
 	installCmd.PersistentFlags().BoolVar(&installCmdOptions.NoWait, "no-wait", false, "Perform non-blocking install")
 	installCmd.PersistentFlags().BoolVarP(&installCmdOptions.Yes, "yes", "y", false, "Do not ask for any confirmation")
+	installCmd.PersistentFlags().BoolVarP(&installCmdOptions.GitOps, "gitops", "g", false, "Prepare output for GitOps")
```

Comment:
Understandable, what would you call it?

Reply:
Some ideas:
- `--prune-metadata` (as suggested in the linked ticket)
- `--prune-extended-fields`
- `--no-extended-fields`
- `--show-extended-fields=false`
- `--only-user-fields`

Reply:
I see, but I don't think these are much more exact/descriptive. Not all metadata is pruned, and the user would have to find out by themselves or by docs what an extended field or a user field is. I believed that we should make it as easy as possible to get going with installing a package for a gitops environment, that's why I wanted to opt for a general "shortcut". I don't know tbh. 

What if we turned it around and make these things invisible by default, and require a flag to include them? That wouldn't help much with the naming, but it would make the gitops use case easier. But I wouldn't do that.

Reply:
we can also change the "pruned" variant to be the default and call the flag `--show-all-fields`

Reply:
We can also merge as is and deprecate the flag later if we decide to change direction here.

Reply:
Changed my mind, I think the pruned default is the best option for now, I'll do that. I was a bit skeptic first because we are not showing the whole truth by default, but I think it's still slightly better for the majority of users.